### PR TITLE
Add kube_node to kubernetes host tags

### PR DESCRIPTION
**Pull request by mfpierre**
### What does this PR do? What is the motivation?
this PR https://github.com/DataDog/datadog-agent/pull/17982 was not reflected in the documentation

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/containers/kubernetes/tag.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ The Agent can attach Kubernetes environment information as "host tags".
   |---------------------|-------------|--------------------------------------------------------|----------------------------------------------------------------|
   | `kube_cluster_name` | Low         | `DD_CLUSTER_NAME` envvar or cloud provider integration | `DD_CLUSTER_NAME` envvar or cloud provider integration enabled |
   | `kube_node_role`    | Low         | Node label `node-role.kubernetes.io/<role>`            | Node label must exist                                          |
+  | `kube_node`         | Low         | `NodeName` field in a pod specifications               |                                                                |
```
Comment (rtrieu) [inline content/en/containers/kubernetes/tag.md:78]:
```suggestion
  | `kube_node`         | Low         | `NodeName` field in a pod's specifications               |                                                                |
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Grammatical feedback but otherwise LGTM

**Review by rtrieu — APPROVED** [review]

**Comment by mfpierre** [comment]
👋 @rtrieu do you happen to know why this one did not merge?

**Comment by rtrieu** [comment]
@mfpierre I'm not sure but I'm trying again and will monitor this one.

**Comment by rtrieu** [comment]
Closing this as I cloned it here: https://github.com/DataDog/documentation/pull/26859

**Pull request closed without merging**